pkg/artifactory/validators.go

```diff
@@ -2,13 +2,14 @@ package artifactory
 
 import (
 	"fmt"
-	"github.com/gorhill/cronexpr"
-	"gopkg.in/ldap.v2"
 	"net/mail"
 	"os"
 	"regexp"
 	"strings"
 
+	"github.com/gorhill/cronexpr"
+	"gopkg.in/ldap.v2"
```

Comment:
This should be at the very end of the import group.

Reply:
this is the wat `make fmt` formatted it. I'd prefer to keep it this way, because next time we format it with `make fmt` it will be changed back.